docs-md/developer-guide/ksqldb-reference/show-queries.md

```diff
@@ -30,15 +30,21 @@ Query Status
 * `ERROR`: the query has entered an error state.
 * `UNRESPONSIVE`: the host running the query returned an error when requesting the query status.
 
+Query Type
+-----------

```

Comment:
cc @AlanConfluent can you review this section? I think we're standardizing on:
- **persistent** queries run on every node and materialize new state
- **push** queries are owned by the client and are terminated when the session ends (I don't think we should document that they only run on a single node, that's not the key characteristic)

Reply:
Yes, I think the definitions you stated here are good ones.  The lifetime I think is one of the most important qualities from a client's perspective as well as the state materialized by persistent queries.  You could call out that persistent queries run until explicitly terminated, that might be too verbose for this purpose.

FYI, I started documenting terminology here, but it's still a work in progress given the changing push query direction so might not be fully up to date with current thinking: https://confluentinc.atlassian.net/wiki/spaces/KSQL/pages/1193742550/ksqlDB+Query+Terminology